### ISSUE TITLE
Action fails on environments that lack the gh cli

### ISSUE THREAD

**Issue by Robinlovelace** [quarto-dev/quarto-actions#5]
As shown here: https://github.com/geocompr/pytest/runs/4988337081?check_suite_focus=true

**Comment by cderv** [comment]
Yes currently the `gh` CLI is a requirement as we need it to download the latest release for Github release page. 

When creating this action, I assumed it would be used in Github Actions runners which all have the `gh` CLI. Wrong assumptiom. 

Using it makes it really easy to get the latest release from the release page. 

Is getting the gh CLI not possible in your case ? 

I can find a workaround but it is less easy - `gh` CLI makes it really convenient. So if it could work for you that is easier. 

I can add a check and a useful error at least. 

What would be the desired behavior for you ? 
* Downloading the gh CLI directly in the action ? 
* Or not using it at all ? 

thanks !

**Comment by Robinlovelace** [comment]
All good questions. 

> Is getting the gh CLI not possible in your case ?

Yes that's possible and what I did. See here: https://github.com/geocompr/pytest/blob/main/.github/workflows/main.yaml#L17-L20

Is there a way to check if the `gh` tool is installed and install it with that if not? Also see the rest of that .yml file, as you'll see after going to the effort of installing the `gh` tool we just proceed to manually install `quarto` with a fixed version!

It's well worth making this action more robust IMO, lots of people will benefit who don't necessarily use vanilla GitHub images for their workflows.

**Comment by cderv** [comment]
> Is there a way to check if the gh tool is installed and install it with that if not?

You mean inside `quarto-install`  action ? I guess we can do that yes. I wasn't sure if it was a good idea to install a tool that the user does not ask for. But I can also uninstall it maybe at the end, like cleaning. 
Maybe that is not easier than not using the `gh` CLI tool for download. 

> we just proceed to manually install quarto with a fixed version!

This is now possible. It has been asked in #6 and just added in #7. You should now be able to do 
````yaml
    steps:
      - uses: quarto-dev/quarto-actions/install-quarto@main
        with:
          version: 0.3.63
````

**Comment by Robinlovelace** [comment]
> This is now possible. It has been asked in #6 and just added in #7. You should now be able to do

Great! And that does not require `gh`?

**Comment by cderv** [comment]
> Great! And that does not require gh?

It still does. But I can make it not use it in this case. Working on it ! 😅

**Comment by Robinlovelace** [comment]
Not tested, but will aim to do so. Many thanks for the fix!

**Comment by Robinlovelace** [comment]
Tested and to my surprise it worked 1st time: https://github.com/geocompr/pytest/commit/6545593c7256a761b0ea6311d279e43bf198ee61

Do you have any other tips on this @cderv ? Many thanks!
